### PR TITLE
#426 - get away from mockables

### PR DESCRIPTION
**Pull request by anandrgitnirman**
Fix the build and get away from mockables 
#429

### CODE REVIEW

File: training/training.proto [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+message ModelId {
+  string model_id = 1;
+  string method_name = 2;
+}
+
+enum Status {
+  CREATED = 0;
+  IN_PROGRESS = 1;
+  UN_KNOW_ERROR =2;
```
Comment (raamb) [inline training/training.proto:12]: Lets just call it ERROR

File: training/training.proto [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+service Model {
+
+  // The AI developer needs to Implement this service and Daemon will call these
+  rpc create_model(CreateModelRequest) returns (ModelDetailsResponse) {}
+  rpc update_model_access(UpdateModelRequest) returns (ModelDetailsResponse) {}
+  rpc delete_model(UpdateModelRequest) returns (ModelDetailsResponse) {}
+  rpc get_model_details(ModelId) returns (ModelDetailsResponse) {}
+  rpc training_status (ModelId) returns (ModelDetailsResponse) {}
```
Comment (raamb) [inline training/training.proto:58]: Can we rename this to "get_training_status"

File: metrics/clients_test.go [hunk 3]
```diff
@@ -53,49 +73,49 @@ func StartMockGrpcService() {
...
 	serviceURL = "localhost:26000"
 	heartbeat, err = callgRPCServiceHeartbeat(serviceURL)
-	assert.True(t, err != nil)
+	assert.True(suite.T(), err != nil)
 }
 
-func Test_callHTTPServiceHeartbeat(t *testing.T) {
-	serviceURL := "http://demo5343751.mockable.io/heartbeat"
+func (suite *ClientTestSuite) Test_callHTTPServiceHeartbeat() {
+	serviceURL := "http://localhost:1111/heartbeat"
```
Comment (raamb) [inline metrics/clients_test.go:93]: Lets make the port dynamic - via config file etc

Comment (anandrgitnirman) [reply metrics/clients_test.go]: have removed the hardcoded endpoint from multiple places, the service url end point is specified as a config file , for a test case , dont wanna add a new config for the port number

Comment (anandrgitnirman) [reply metrics/clients_test.go]: [e949ae6](https://github.com/singnet/snet-daemon/pull/566/commits/e949ae6e5b0789b26553a46c7d341f25a6cb082e) - latest commit

**Review by raamb — APPROVED** [review]

**Pull request merged**